src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -208,15 +209,17 @@ def _check_compliance_gui_attributes(df: pd.DataFrame) -> dict[str, pd.Series] |
         case None, None:
             return None
         case pd.Series(), pd.Series():
-            final_series = pd.Series(np.logical_or(mandatory_check, no_attribute_check))  # type: ignore[arg-type]
+            mandatory_check = cast("pd.Series[bool]", mandatory_check)
+            no_attribute_check = cast("pd.Series[bool]", no_attribute_check)
+            final_series: pd.Series[bool] = pd.Series(np.logical_or(mandatory_check, no_attribute_check))
         case pd.Series(), None:
-            final_series = mandatory_check
+            final_series = cast("pd.Series[bool]", mandatory_check)
         case None, pd.Series:
-            final_series = no_attribute_check
+            final_series = cast("pd.Series[bool]", no_attribute_check)
```

Comment:
I understand in theory why more typing is good, but this makes the code extremely verbose and in my opinion harder to understand, I appreciate this form of extreme typing if the datatype is very unclear but in this case there is no other datatype possible for `no.logical_or` therefore that is not an advantage. So all in all, in my opinion in this case, this does not improve the code.

Reply:
I agree it's ugly. The only alternative I see is to add `# type: ignore` comments, which I did now.

Reply:
Much appreciated thanks.